### PR TITLE
Add image gallery / InlinePanel section to tutorial

### PR DESCRIPTION
**Pull request by gasman**
Addresses #3148. cc @shacker

### CODE REVIEW

**Comment by shacker** [comment]
This is excellent, and very clear @gasman - and great timing as I'm tasked with building a carousel feature today.

**Comment by shacker** [comment]
Worked perfectly for me, except that I had to `{% load  wagtailimages_tags %}`. But that is mentioned earlier in the tutorial, so if a person keeps working on the same page, all set.

File: docs/getting_started/tutorial.rst [hunk 1]
```diff
@@ -416,14 +413,41 @@ your ``BlogPage`` model:
...
 Run ``python manage.py makemigrations`` and ``python manage.py migrate``.
 
-Adjust your blog page template to include the image:
+There are a few new concepts here, so let's take them one at a time:
+
+Inheriting from ``Orderable`` adds a ``sort_order`` field to the model, to keep track of the ordering of images in the gallery.
+
+The ``ParentalKey`` to ``BlogPage`` is what attaches the gallery images to a specific page. A ``ParentalKey`` works similarly to a ``ForeignKey``, but also defines ``BlogPageGalleryImage`` as a "child" of the ``BlogPage`` model, so that it's treated as a fundamental part of the page in operations like submitting for moderation, and tracking revision history.
+
+``image`` is a ``ForeignKey`` to Wagtail's built-in ``Image`` model, where the images themselves are stored. This comes with a dedicated panel type, ``ImageChooserPanel``, which provides a popup interface for choosing an existing image or uploading a new one. This way, we allow an image to exist in multiple galleries - effectively, we've created a many-to-many relationship between pages and images.
```
Comment (gasman) [inline docs/getting_started/tutorial.rst]: Fine by me! Fixed in cdf8c52faac137d092ad05815648948ac39810d7.

**Comment by gasman** [comment]
@shacker As far as I can see we've got the `{% load wagtailimages_tags %}` on all the template examples that need it. Agreed that it's easy to overlook, though - I also missed it on my last run through, which is what prompted me to add the note in the text that specifically calls it out. Perhaps there's more we can do to highlight this... although I'm inclined to say that the experience of missing it out, getting the error and tracking down the bug is _probably_ a useful learning tool for people following the tutorial!

**Comment by gasman** [comment]
Merged in 3fd5bb1820f17b5e9bf8b67a8761608f0037c3d5

**Pull request closed without merging**